packages/ilmomasiina-models/src/schema/event/attributes.ts

```diff
@@ -4,12 +4,18 @@ import { Nullable } from "../utils";
 
 export const eventSlug = Type.String({
   description: "Event slug, used for accessing the event by URL.",
+  minLength: 1,
+  maxLength: 255,
```

Comment:
Same for this, is a 255 character slug too long cause then you have the hash on top for signup edits?

Reply:
The event slug is not present in signup edits